src/wrappers/withDragHandles.js

```diff
@@ -84,7 +84,18 @@ export function dragHandle(handle) {
         // preventing default to prevent lag on touch devices (because of the browser checking for screen scrolling)
         e.preventDefault();
         isItemsDragDisabled.set(false);
+
+        // Reset isItemsDragDisabled if the user releases the mouse/touch without initiating a drag
+        const onEnd = () => {
```

Comment:
I wonder if this could simply replace the finalize function.
If not please use 'function onEnd' instead of 'const onEnd = () =>' for consistency and consider pulling it out as it manipulates the store, not a function scoped variable